.pyinstaller/run_sunpy_tests.py

```diff
@@ -0,0 +1,81 @@
+import os
+import sys
+import shutil
+
+import pytest
+
+# Skipping these tests that take check the name of the current module (which ends up starting with
+# sunpy_tests rather than sunpy).
+# asdf path issue with PyInstaller as well.
```

Comment:
Is this an asdf issue or a sunpy issue?

Reply:
An issue specifically with asdf and PyInstaller, works fine when used without. 
The other tests take the wrong directory into consideration.

Reply:
Is this just the tests, or will asdf fail to find the schemas and break when running user code?

Reply:
Just these tests from what I've noticed. I haven't checked it thoroughly though, is there any way we can?